_life/money-americans.md

```diff
@@ -423,7 +434,14 @@ boy can things get complicated real fast._
       with _debit cards_, which are radically different, which just goes to show how
       deep the non-credit card culture is.
 
-[^2]: It is only the case for someone moving in from a different canton within Switzerland.
+[^2]: There is some nuance here. If you are moving into Vaud from abroad (but not
+      from a different Swiss canton) partway through a year, the tax _rate_ for your
+      Swiss income is determined by proportionally scaling that income to the whole
+      year, and this is reflected in the source deductions you will see on your
+      paystubs. But there is an automatic 16,000 CHF deduction from your taxable
+      income for the benefit of _contribuables modestes_ (low-income taxpayers) which
+      means that with an EPFL PhD student's salary, your taxable income for the first
+      (partial) year will be zero.
```

Comment:
@SolalPirelli I amend what I said to you earlier about first-year TOUs with this new knowledge.

Reply:
ok, that does make more sense to me based on the little data I have ^^ thanks!